src/test/java/uk/gov/hmcts/darts/audio/service/impl/AudioOperationServiceImplTest.java

```diff
@@ -45,113 +48,123 @@ class AudioOperationServiceImplTest {
+                1
+            )
+        );
     }
 
     @Test
     void shouldGenerateConcatenateCommandWhenValidAudioFilesAreReceived() {
         when(audioConfigurationProperties.getFfmpegExecutable()).thenReturn(FFMPEG);
 
-        CommandLine expectedCommand = CommandLine.parse(
-            "/usr/bin/ffmpeg -i /path/to/audio/requestId/sample1-5secs.mp2 -i /path/to/audio/requestId/sample2-5secs.mp2"
-                + " -filter_complex \"[0:a][1:a]concat=n=2:v=0:a=1\" /tempDir/concatenate/requestId/C1-concatenate-20230510145233697.mp2");
+        List<AudioFileInfo> inputAudioFileInfos = List.of(
+            new AudioFileInfo(
+                Instant.parse(T_09_00_00_Z),
+                Instant.parse(T_10_30_00_Z),
+                "/path/to/audio/original0.mp3",
+                1
+            ),
+            new AudioFileInfo(
+                Instant.parse(T_10_30_00_Z),
+                Instant.parse(T_11_00_00_Z),
+                "/path/to/audio/original1.mp3",
+                1
+            )
+        );
 
-        CommandLine concatenateCommand = audioOperationService.generateConcatenateCommand(
-            1,
-            audioFileInfos,
-            "/tempDir/concatenate/requestId/C1-concatenate-20230510145233697.mp2"
+        CommandLine actualCommand = audioOperationService.generateConcatenateCommand(
+            inputAudioFileInfos,
+            Path.of("/path/to/output/audio.mp2")
         );
 
-        assertNotNull(concatenateCommand);
-        assertEquals(expectedCommand.getArguments().length, concatenateCommand.getArguments().length);
-        assertEquals(expectedCommand.getExecutable(), concatenateCommand.getExecutable());
-        assertEquals(expectedCommand.toString(), concatenateCommand.toString());
+        CommandLine expectedCommand = CommandLine.parse(
+            "/usr/bin/ffmpeg -i /path/to/audio/original0.mp3 -i /path/to/audio/original1.mp3"
+                + " -filter_complex [0:a][1:a]concat=n=2:v=0:a=1 /path/to/output/audio.mp2");
+
+        assertNotNull(actualCommand);
+        assertEquals(expectedCommand.toString(), actualCommand.toString());
     }
 
     @Test
     void shouldReturnConcatenatedAudioFileInfoWhenValidInputAudioFiles() throws Exception {
         when(audioConfigurationProperties.getFfmpegExecutable()).thenReturn(FFMPEG);
-        when(audioConfigurationProperties.getConcatWorkspace()).thenReturn("/tempDir/concatenate");
+        when(audioConfigurationProperties.getConcatWorkspace()).thenReturn(tempDirectory.toString());
         when(systemCommandExecutor.execute(any())).thenReturn(Boolean.TRUE);
 
-        AudioFileInfo expectedAudio = new AudioFileInfo(
-            Instant.parse(T_09_00_00_Z),
-            Instant.parse(T_11_00_00_Z),
-            "/tempDir/concatenate/requestId/C1-concatenate-20230510145233697.mp2",
-            1
+        AudioFileInfo audioFileInfo = audioOperationService.concatenate(
+            WORKSPACE_DIR,
```

Comment:
Should this test example make use of something similar to `UUID.randomUUID().toString()` since we've chosen at this point to not make a subfolder with the `requestId`?

Reply:
Good point, will update.